src/service/video-manager-impl.js

```diff
@@ -325,6 +364,44 @@ class VideoEntry {
   }
 
   /**
+   * Called when the orientation of the device changes
+   * @private
+   */
+  orientationChanged_() {
+
+    const vidElement = this.video.element.querySelector('iframe, video');
+
+    // Determine which implementation of the orientation angle to use
+    const angle = this.ampdoc_.win.orientation !== undefined ?
+                  this.ampdoc_.win.orientation : screen.orientation.angle;
+
+    if (this.isVisible_
+    && this.getPlayingState() == PlayingStates.PLAYING_MANUAL
+    && (Math.abs(angle) == 270 || Math.abs(angle) == 90)) {
+
+      // Determine which implementation of requestFullScreen to use
+      const requestFullScreen = vidElement.webkitEnterFullscreen
+      || vidElement.webkitRequestFullScreen || vidElement.requestFullscreen
+      || vidElement.webkitEnterFullscreen || vidElement.msRequestFullscreen
+      || vidElement.mozRequestFullScreen;
+
+      requestFullScreen.call(vidElement);
+      this.isFullscreen_ = true;
+
+    } else if (this.isFullscreen_) {
```

Comment:
`this.isFullscreen_` is misleading, it is more like `this.isFullscreenWithAutoFullscreen_`